src/components/CardTabs/Users/index.js

```diff
@@ -90,8 +100,13 @@ const UserCard = (props) => {
                   </Box>
                 </Box>
                 <Box
-                  onClick={() => user.onClick}
+                  onClick={e => {
+                    handleOnClick(e);
+                  }}
                   data-testId={index == 0 ? "UserAdd" : ""}
+                  sx={{
+                    cursor: "pointer"
+                  }}
                 >
                   <img src={AddUser} />
```

Comment:
Add conditional in `<img />` tag itself.
Handle the state using `useState` hook.

Currently the handle click function is adding up extra unnecessary complexity.

Reply:
create a state inside the <UserElement /> instead. inside the component

Reply:
yes